File: Cs50x/Week[1]/MarioAdvanced.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
      
    int deger;
    do
    {
        deger= get_int("1 ile 8 arasında bir rakam gir!");
    }

    while (deger<1 || deger>8);
           
   for(int i = 0; i <deger; i++)
   {
        for(int k=0;k<deger-i-1;k++)
       {
            printf(" ");
       } 
       for(int j = 0; j <= i; j++)
       {
           printf("#");
           
       }
       printf("  ");
        for(int j=0;j<=i;j++){
            printf("#");
        }
       printf("\n");
   }
}   

File: Cs50x/Week[6]/MarioAdvanced.py
from cs50 import get_int


yukseklik=get_int("Height:")
while yukseklik<1 or yukseklik>8:
    yukseklik=get_int("Height:")
        

for i in range(0,yukseklik):
    for k in range(0,yukseklik-i-1):
        print(" ",end = "")
    for j in range(0,i+1):
        print("#", end = "" )
        
    print("  ", end = "")   
    for j in range(0,i+1):
        print("#", end = "" )
    print("")
